2022-10-16/src/slides.md

```diff
@@ -273,9 +273,10 @@ function unref(input) {
 Which means:
 
 ```ts
-const foo = unref(123)
+const foo = 123
+const bar = ref(foo)
 
-unref === 123 // true
+unref(foo) === unref(bar) // true
```

Comment:
```suggestion
foo === 123 // true
```

It's actually a simple typo 😅

Reply:
-0-  🤗 Sorry for committing several times. I'm not much familiar with GH pr workflows.